allen-cell-animated/simularium-website
# Feature/update viewer

Problem
=======
We need to update the viewer to use user colors

Solution
========
Update the viewer, and use colors in UI display data. 

## Type of change
Please delete options that are not relevant.
* New feature (non-breaking change which adds functionality)
* This change requires updated or new tests

Change summary:
---------------
* Updated viewer
* Added color to test data
* Use display data color for color swatches

Steps to Verify:
----------------
1. npm start
2. drag and drop one of the v3 example files 

Screenshots (optional):
-----------------------
<img width="1346" alt="Screen Shot 2021-10-18 at 3 44 47 PM" src="https://user-images.githubusercontent.com/5170636/137816499-3eb78576-9361-4434-8dea-6f5e61021eb3.png">
